src/main/java/study/step3/RacingGame.java

```diff
@@ -0,0 +1,61 @@
+package study.step3;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+public class RacingGame {
+    private final static int INIT_INDEX = 0;
+    private final static int FIRST_DISTINCTION_NUMBER = 1;
+    private final static Pattern PATTERN = Pattern.compile("^[0-9]+$");
+    private final static String ILLEGALARGUMENT_EXCEPTION_MASSEAGE = "올바르지 않은 값을 입력했습니다.";
+
+    private int time= 0;
+    private int carNumbers = 0;
+    private List<Car> carList;
```

Comment:
```suggestion
    private int time;
    private int carNumbers;
    private List<Car> carList = new ArrayList<>();
```
변수명에 타입을 사용하지 않도록 하는건 어떨까요 ? 
carList 가 set으로 변경된다면  carList를 사용하는 모든 코드에 수정이 일어 날것 같아요.